File: Leetcode/python/sol3340.py
# https://leetcode.com/problems/check-balanced-string/description/

class Solution:
    def isBalanced(self, num: str) -> bool:
        suma: int = 0
        for i, j in enumerate(num):
            if i % 2 == 0:
                suma += int(j)
            else:
                suma -= int(j)
        return not suma

File: Leetcode/javascript/sol3340.js

//  https://leetcode.com/problems/check-balanced-string/

/**
 * @param {string} num
 * @return {boolean}
 */
var isBalanced = function(num) {

    var suma = 0;

    for(let i = 0; i<num.length; i++){
        if(i%2 == 0){
            suma+=parseInt(num[i]);
        }else{
            suma-=parseInt(num[i]);
        }
    }

    return !suma;

};